openwillis/measures/text/util/characteristics_util.py

```diff
@@ -1055,346 +637,327 @@ def update_summ_df(
 
     ------------------------------------------------------------------------------------------------------
     """
-    if len(phrase_df) > 0:
-        speech_minutes = phrase_df[measures["phrase_minutes"]].sum()
+    if len(turn_df) > 0:
+        speech_minutes = turn_df[measures["turn_minutes"]].sum()
     else:
         speech_minutes = (float(df_diff.iloc[-1][time_index[1]]) - float(df_diff.iloc[0][time_index[0]])) / 60
-
     summ_df[measures["speech_minutes"]] = [speech_minutes]
-
+    
     summ_df[measures["speech_words"]] = len(df_diff)
     if speech_minutes > 0:
-        summ_df[measures["word_rate"]] = (
-            summ_df[measures["speech_words"]] / summ_df[measures["speech_minutes"]]
-        )
-        summ_df[measures["syllable_rate"]] = (
-            get_num_of_syllables(full_text) / summ_df[measures["speech_minutes"]]
-        )
+        
+        summ_df[measures["word_rate"]] = (summ_df[measures["speech_words"]] / summ_df[measures["speech_minutes"]])
+        summ_df[measures["syllable_rate"]] = (get_num_of_syllables(full_text) / summ_df[measures["speech_minutes"]])
+        
         summ_df[measures["speech_percentage"]] = 100 * (
-        1
-        - df_diff.loc[1:, measures["pause"]].sum()
-        / (60 * summ_df[measures["speech_minutes"]])
-    )
+        1 - df_diff.loc[1:, measures["pause"]].sum()/ (60 * summ_df[measures["speech_minutes"]]))
```

Comment:
```suggestion
        summ_df[measures["speech_minutes"]] / summ_df[measures["file_length"]])
```

That way works for multiple speakers as well